x/examples/fetch/main.go

```diff
@@ -52,8 +55,24 @@ func init() {
 	}
 }
 
+func overrideAddress(original string, newHost string, newPort string) (string, error) {
+	host, port, err := net.SplitHostPort(original)
+	if err != nil {
+		return "", fmt.Errorf("invalid address: %w", err)
+	}
+	if newHost != "" {
+		host = newHost
+	}
+	if newPort != "" {
+		port = newPort
+	}
+	return net.JoinHostPort(host, port), nil
+}
+
 func main() {
 	verboseFlag := flag.Bool("v", false, "Enable debug output")
+	tlsKeyLogFlag := flag.String("tls-key-log", "", "Filename to write the TLS key log to allow for decryption on Wireshark.")
```

Comment:
*nit* Remove the '.' since other options do not end with '.'.

```suggestion
	tlsKeyLogFlag := flag.String("tls-key-log", "", "Filename to write the TLS key log to allow for decryption on Wireshark")
```